sustc-api/src/main/java/io/sustc/dto/UserRecord.java

```diff
@@ -1,44 +1,70 @@
 package io.sustc.dto;
 
-import com.opencsv.bean.CsvBindByName;
 import lombok.Data;
 
 @Data
 public class UserRecord {
 
-    @CsvBindByName(column = "Mid")
-    private Long mid;
+    /**
+     * The user's ID
+     */
+    private long mid;
 
-    @CsvBindByName(column = "Name")
+    /**
+     * The user's name
+     */
     private String name;
 
-    @CsvBindByName(column = "Sex")
+    /**
+     * The user's sex
+     */
     private String sex;
 
-    @CsvBindByName(column = "Birthday")
+    /**
+     * The user's birthday
+     */
     private String birthday;
 
-    @CsvBindByName(column = "Level")
-    private Short level;
+    /**
+     * The user's level
+     */
+    private int level;
 
-    @CsvBindByName(column = "Sign")
-    private String sign;
+    /**
+     * The user's current number of coins
+     */
+    private int coin;
 
-    @CsvBindByName(column = "following")
-    private Long[] following;
+    /**
+     * The user's personal sign, can be empty
+     */
+    private String sign;
 
-    @CsvBindByName(column = "identity")
+    /**
+     * The user's identity
+     */
     private Identity identity;
 
-    @CsvBindByName(column = "password")
+    /**
+     * The user's password
+     */
     private String password;
 
-    @CsvBindByName(column = "qq")
+    /**
+     * The user's qq
+     */
     private String qq;
 
-    @CsvBindByName(column = "wechat")
+    /**
+     * The user's wechat
```

Comment:
may be null/empty? (check my comment on the field `sign`)

Reply:
Since all corner cases note that an empty field can be null or empty, I think that statement `can be null or empty` is enough. Specification may lead to more problems in implementation since they may forget to deal with some cases while themselves are testing these cases.